run.py

```diff
@@ -45,12 +45,23 @@ def run(command, env={}, ignore_errors=False):
                    'provided all subjects should be analyzed. Multiple '
                    'participants can be specified with a space separated list.',
                    nargs="+")
+parser.add_argument('--session_label', help='The label of the session that should be analyzed. The label '
+                   'corresponds to ses-<session_label> from the BIDS spec '
+                   '(so it does not include "ses-"). If this parameter is not '
+                   'provided all sessions should be analyzed. Multiple '
+                   'sessions can be specified with a space separated list.',
+                   nargs="+")
 parser.add_argument('--n_cpus', help='Number of CPUs/cores available to use.',
                    default=1, type=int)
 parser.add_argument('--stages', help='Autorecon stages to run.',
                     choices=["autorecon1", "autorecon2", "autorecon2-cp", "autorecon2-wm", "autorecon-pial", "autorecon3", "autorecon-all", "all"],
                     default=["autorecon-all"],
                     nargs="+")
+parser.add_argument('--steps', help='Longitudinal steps to run.',
```

Comment:
Could we use human readable labels for the steps instead of numbers? It would be easier to understand what this option is for. Maybe: 'session', 'subject_template', 'long' ?